lib/cadet/assessments/submission.ex

```diff
@@ -21,15 +17,9 @@ defmodule Cadet.Assessments.Submission do
     timestamps()
   end
 
-  @required_fields ~w(status)a
-  @optional_fields ~w(override_xp submitted_at)a
-
   def changeset(submission, params) do
-    params = convert_date(params, :submitted_at)
-
     submission
-    |> cast(params, @required_fields ++ @optional_fields)
-    |> validate_required(@required_fields)
+    |> cast(params, [])
     |> validate_role(:student, :student)
```

Comment:
We're moving role validation up to context right? If so maybe can just remove these

Reply:
these are removed in my branch. I don't want to touch it here.